variables.tf

```diff
@@ -165,6 +165,18 @@ variable "olm_namespace" {
   type        = string
 }
 
+variable "olm_use_istio" {
+  default     = false
+  description = "Apply Istio authorization policies for OLM operators. Defaults to \"false\"."
+  type        = bool
+}
+
+variable "olm_istio_root_namespace" {
```

Comment:
Should this one be the same value as `istio_system_namespace` used in https://github.com/streamnative/terraform-helm-charts/blob/master/modules/istio-operator/main.tf#L56 ?

Reply:
To be honest, "root namespace" is the ideal term because Istio uses this term ([docs](https://istio.io/latest/search/?q=root%20namespace)).  Of course the default value is `istio-system`, so either are pretty good.  I chose to use `istio_system_namespace`.